docs/theme/included/powerline.rst

```diff
@@ -334,6 +334,12 @@ ______
 
    Color for the chroot section.
 
+.. attribute:: POWERLINE_DIRSTACK_COLOR
+   :type: array<int>
+   :value>: :attr:`POWERLINE_NEUTRAL_COLOR`
```

Comment:
Ugh, the ReadTheDocs build failed, so I missed this error. Glad I decided to build the docs manually before merging. Please amend this:
```suggestion
   :value: :attr:`POWERLINE_NEUTRAL_COLOR`
```
This has to be the last thing I ask for.

Reply:
Oh my 🤦🏻 Sorry 😄